samples/stock/navigator/sticktomax/demo.details

```diff
@@ -0,0 +1,6 @@
+---
+ name: Navigator stickToMax false
```

Comment:
Thank you! I haven't created a demo for `stickToMax-true` because it's the default behavior of the navigator, and it's quite intuitive in my opinion. Requests on the forum were mostly about disabling this feature, so I thought that the API property description will be sufficient for the users. But it's your call!

Reply:
> I haven't created a demo for stickToMax-true because it's the default behavior of the navigator,

That's not entirely true. The default is `undefined`, which means it sticks to max _only_ if it was already at max prior to adding points. When setting `stickToMax: true`, the navigator will move to the max also if it wasn't already stuck to max from before.

Did you consider that? I'm happy with the state of this PR now, just want to confirm that this is the intended behaviour.

Reply:
Thank you for the clarification, now I understand the difference. The current implementation is the intended behaviour, so I think we can close this PR if you are happy with it.